### DISCUSSION TITLE
Unicorn Ultra x Akash Network

### DISCUSSION THREAD

**Discussion by haibx-uniultra** [akash-network/community#602] [Integrations]
Hi all, Unicorn Ultra is looking for a possible collaboration for Akash Network to be built on our chain. We think that you would fit very well in our ecosystem, which is Depin-centric at the moment. Our team also provides grants for suitable projects as well when they are built on our chain. Let me know your thoughts regarding this

**Comment by 31trainman** [comment]
What on akash network are you looking to integrate with exactly?

**Reply by haibx-uniultra** [reply]
We have not had a clear picture yet, but seeing that we are both Layer 1 within the Depin Marketspace, the first step can be Marketing Partnership. Afterward, I would love to connect our ecosystem team and your team to discuss more on ways for technical integration.

**Reply by 31trainman** [reply]
Send me a discord friend request. My discord is 31trainman.

**Reply by haibx-uniultra** [reply]
I have just sent a request, my discord is ted1312

**Comment by ghost** [comment]
What kind of collaboration?
